From ba2903c4c8fb5879cea8ae4c0eda6bd6e4b9ac34 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:12:34 -0400
Subject: [ethernet] i40e/i40evf: remove reserved type

Message-id: <1409663591-32531-112-git-send-email-sassmann@redhat.com>
Patchwork-id: 90591
O-Subject: [RHEL7.1 PATCH 111/148] i40e/i40evf: remove reserved type
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jesse Brandeburg <jesse.brandeburg@intel.com>

One of the PCTYPES that was moved to a reserved value
wasn't removed from the code.

Change-ID: I31fafe6d79c5f5128179979af5eaafa8c0cd62fe
Signed-off-by: Jesse Brandeburg <jesse.brandeburg@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit c571ea05a03da856b1e9f8c0355128a7044d6a91)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40e_txrx.h b/drivers/net/ethernet/intel/i40evf/i40e_txrx.h
index 30d248b..acd3c12 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_txrx.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_txrx.h
@@ -75,7 +75,6 @@ enum i40e_dyn_idx_t {
  ((u64)1 << I40E_FILTER_PCTYPE_NONF_IPV4_OTHER) | \
  ((u64)1 << I40E_FILTER_PCTYPE_FRAG_IPV4) | \
  ((u64)1 << I40E_FILTER_PCTYPE_NONF_IPV6_UDP) | \
- ((u64)1 << I40E_FILTER_PCTYPE_NONF_IPV6_TCP_SYN) | \
  ((u64)1 << I40E_FILTER_PCTYPE_NONF_IPV6_TCP) | \
  ((u64)1 << I40E_FILTER_PCTYPE_NONF_IPV6_SCTP) | \
  ((u64)1 << I40E_FILTER_PCTYPE_NONF_IPV6_OTHER) | \
diff --git a/drivers/net/ethernet/intel/i40evf/i40e_type.h b/drivers/net/ethernet/intel/i40evf/i40e_type.h
index 9c83578..23cd18b 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -878,7 +878,6 @@ enum i40e_filter_pctype {
  I40E_FILTER_PCTYPE_FRAG_IPV4   = 36,
  /* Note: Values 37-40 are reserved for future use */
  I40E_FILTER_PCTYPE_NONF_IPV6_UDP  = 41,
- I40E_FILTER_PCTYPE_NONF_IPV6_TCP_SYN  = 42,
  I40E_FILTER_PCTYPE_NONF_IPV6_TCP  = 43,
  I40E_FILTER_PCTYPE_NONF_IPV6_SCTP  = 44,
  I40E_FILTER_PCTYPE_NONF_IPV6_OTHER  = 45,
-- 
1.7.1